system/lib/sbrk.c

```diff
@@ -99,6 +101,11 @@ void *sbrk(intptr_t increment) {
 }
 
 int brk(intptr_t ptr) {
+#if __EMSCRIPTEN_PTHREADS__
```

Comment:
I guess we could even make this work only on the main thread (but not background threads) if we wanted.